LF: force `actor` field in Exercise Update to be undefined. (#7578)
digital-asset/daml: daml-lf/spec/daml-lf-1.rst

  + **Add** ``COERCE_CONTRACT_ID`` primitive for coercing ``ContractId``.

>   + **Change** ``Update.Exercise`` such that ``actor`` is dropped.

Reviewer: ```suggestion
  + **Change** ``Update.Exercise`` such that ``actor`` must not be set anymore.
```
